chain/src/test/java/org/ethereum/beacon/chain/DefaultBeaconChainTest.java

```diff
@@ -30,10 +33,12 @@
 
   @Test
   public void insertAChain() {
-    SpecHelpers specHelpers = SpecHelpers.createDefault();
+    Schedulers schedulers = new DefaultSchedulers();
```

Comment:
What if we keep `Schedulers.getDefault()` to instantiate a `DefaultSchedulers` object? First, this is convenient, second, it would be easier to swap a default implementation of `Schedulers` later on.

Reply:
Third, it increases UX of `Schedulers`, user would be able to pick some default implementation without additional investigation.

Reply:
Yep, I was going to extract `Schedulers` interface a bit later, but let's do it in this PR...

Reply:
**EDT:** `Schedulers.createDefault()` is better.